Raven.Database/Bundles/Replication/Impl/ReplicationHiLo.cs

```diff
@@ -32,15 +32,9 @@ private ReplicationHiLo(DocumentDatabase database)
             var document = database.Documents.Get(RavenReplicationVersionHiLo, null);
             //we do not have the hilo document (old format), 
             if (document == null) 
-            {
-                Database.TransactionalStorage.Batch(accessor =>
```

Comment:
@myarichuk why have you reverted this ?

Reply:
@talweiss1982 
I know that it was better this way, but I had a good reason to revert:
A call to GetNextIdentityValue() increments the record version, and in some cases it creates concurrency exception when trying to write *hilo* records. 
At least one test failed because of it (related to smuggler writing the hilos)